### PR TITLE
PLENIGO-749-test-max-title-length-in-php

### PR DESCRIPTION
**Pull request by SebaSOFT**
- Added validation to the title length and product ID length
- Added test for that

### CODE REVIEW

File: src/plenigo/internal/server-interface/payment/Checkout.php [hunk 1]
```diff
@@ -29,8 +33,13 @@
...
  * @link     https://www.plenigo.com
  */
-final class Checkout extends ServerInterface
-{
+final class Checkout extends ServerInterface {
+
+    const TITLE_MAX_LENGTH = 100;
+    const PROD_ID_MAX_LENGTH = 20;
+    
+    const ERR_MSG_TITLE_TOO_LONG = "The Product title is too long and it will be truncated (10 chars max.)";
```
Comment (maximilianschweitzer) [inline src/plenigo/internal/server-interface/payment/Checkout.php]: 100 chars max

File: src/plenigo/internal/server-interface/payment/Checkout.php [hunk 2]
```diff
@@ -29,8 +33,13 @@
...
+    const ERR_MSG_PROD_ID_TOO_LONG = "The Product ID must be up to 20 chars long!";
```
Comment (maximilianschweitzer) [inline src/plenigo/internal/server-interface/payment/Checkout.php]: can be up

File: src/plenigo/internal/server-interface/payment/Checkout.php (hunk 1 above)
Comment (SebaSOFT) [inline src/plenigo/internal/server-interface/payment/Checkout.php]: done

File: src/plenigo/internal/server-interface/payment/Checkout.php (hunk 2 above)
Comment (SebaSOFT) [inline src/plenigo/internal/server-interface/payment/Checkout.php]: done

**Pull request merged**